server/application-server/src/main/resources/application.yml

```
@@ -51,6 +51,13 @@ hephaestus:
     intelligence-service:
         url: http://localhost:8000
 
+    cache:
+        contributors:
+                cache:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 16

(indentation)